src/Components/Nav/Nav.scss

```
@@ -9,4 +11,100 @@
   box-shadow: rgba(0, 0, 0, 0.08) 0px 1px 0px 0px;
   background-color: $white;
   transition: background-color 200ms ease 0s;
-}
+
+  .navContainer {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 1320px;
+
+    .navLeft,
+    .navRight {
+      display: flex;
+      align-items: center;
+    }
+
+    li {
+      list-style-type: none;
+
+      .logo {
+        width: 160px;
+        cursor: pointer;
+      }
+
+      .movieBtn,
+      .tvBtn,
+      .bookBtn,
+      .loginBtn {
+        cursor: pointer;
+        margin-left: 24px;
+        font-size: 15px;
+        color: rgb(126, 126, 126);
+        background: none;
+        border: none;
+
+        &:focus {
+          outline: none;
+        }
+      }
+
+      .searchContainer {
+        @include center;
+        width: 300px;
+        height: 62px;
+
+        .label {
+          @include center;
+          width: 300px;
+          height: 38px;
+          padding: 0 10px 0 36px;
+          background-color: rgb(245, 245, 247);
+          background-image: url("/images/nav/search_icon.svg");
+          background-size: 16px;
+          background-repeat: no-repeat;
+          background-position: 4% 50%;
+
+          input {
+            width: 254px;
+            height: 24px;
+            font-size: 14px;
+            font-weight: 400;
+            letter-spacing: -0.3px;
+            width: 100%;
+            border: none;
+            background: none;
+
+            &:focus {
+              outline: none;
+            }
+          }
+
+        }
+
+      }
+    }
+
+    .signUpBtn {
+      cursor: pointer;
+      text-align: center;
+      font-weight: 500;
+      line-height: 20px;
+      box-sizing: border-box;
+      width: auto;
+      min-width: 72px;
+      height: 32px;
+      color: rgb(53, 53, 53);
+      font-size: 14px;
+      letter-spacing: -0.3px;
+      border-radius: 6px;
+      background: transparent;
+      padding: 5px 14px 6px;
+      border: 1px solid rgba(116, 116, 123, 0.5);
+      margin-left: 24px;
```

```suggestion
      background: transparent;
      padding: 5px 14px 6px;
      margin-left: 24px;
      border: 1px solid rgba(116, 116, 123, 0.5);
      border-radius: 6px;
```
유사한 속성끼리 묶어서 배치해주세요.

---

넵